package-lock.json

```diff
@@ -12677,8 +12677,7 @@
         "ansi-regex": {
           "version": "2.1.1",
           "bundled": true,
-          "dev": true,
-          "optional": true
+          "dev": true
```

Comment:
Are changes in `pacakge-lock.json` with no changes in `package.json` intentional?

Reply:
No. I'll remove this from the PR